ibm_was/datadog_checks/ibm_was/data/conf.yaml.example

```diff
@@ -0,0 +1,60 @@
+init_config:
+
+instances:
+  ## @param servlet_url - string - required
+  ## The URL of the PerfServlet configured to report metrics
+  #
+  - servlet_url: http://localhost/wasPerfTool/servlet/perfservlet
+
+  ## @param skip_proxy - boolean - optional - default: False
+  ## Wheter or not to skip the proxy configured in the Agent's config file when
+  ## making the request to the PerfServlet 
+  #
+  # skip_proxy: False
+
+  ## @param collect_jdbc_stats - boolean - optional - default: True
+  ## Whether or not to collect JDBC Connection Pool stats
+  #
+  # collect_jdbc_stats: True
+
+  ## @param collect_jvm_stats - boolean - optional - default: True
+  ## Whether or not to collect JVM Runtime stats
+  #
+  # collect_jvm_stats: True
+
+  ## @param collect_servlet_session_stats - boolean - optional - default: True
+  ## Whether or not to collect Servlet Session Manager stats
+  #
+  # collect_servlet_session_stats: True
+
+  ## @param collect_thread_pool_stats - boolean - optional - default: True
+  ## Whether or not to collect Thread Pools stats
+  #
+  # collect_thread_pool_stats: True
+
+  ## @param custom_queries - custom - optional
+  ## List of custom or additional XML attributes to parse and submit as metrics
+  ## Each must contain a `metric_prefix` and `stat` element
+  ## If the `stat` node has sub `stat` nodes, the name of these nodes must be specified
+  ## in the order they occur within the XML document and will be displayed as tags.
```

Comment:
Is the order mandatory ? Are you sure the XML document will always have the same order ? Could we instead have a way to not care about the order ?

Reply:
To be clear, I only mean the order in which elements are nested. If elements have the same parent, the check works independent of the order. Its only making the assumption that the order of the children hierarchy remain the same. I.e. A always has child B always has child C. If A has children B and C on the same level, the check is OK there. Let me know if that makes sense 😄

Reply:
Ok it makes more sense thanks :)
Could you maybe add an example of a custom config where you have to respect the order ? That would help as well.